extensions/amp-date-countdown/validator-amp-date-countdown.protoascii

```diff
@@ -36,6 +36,10 @@ tags: {  # <amp-date-countdown>
     value_casei: "minutes"
     value_casei: "seconds"
   }
+  attrs {
+    name: "count-up"
```

Comment:
Please leave a comment that this is noop in 0.1

Reply:
Great, added!